src/sentry/snuba/discover.py

```diff
@@ -796,8 +799,10 @@ def get_performance_facets(
             return []
 
     results = []
+    snuba_filter.conditions.append([aggregate_column, "IS NOT NULL", None])
 
-    sampling_enabled = True
+    # Only enable sampling if over 10000 values
```

Comment:
out of curiosity was 10k chosen arbitrarily?

Reply:
Same as above, presumably yes. Ideally we can set it to hit a target confidence rate, but that can be for later.